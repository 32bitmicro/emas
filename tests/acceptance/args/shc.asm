shc r0, 1
shc r1, 2
shc r2, 3
shc r3, 4
shc r4, 5
shc r5, 6
shc r6, 7
shc r7, 8
shc r6, 9
shc r5, 10
shc r4, 11
shc r3, 12
shc r2, 13
shc r1, 14
shc r0, 15
shc r1, 0
shc r2, -1
shc r3, -2
shc r4, -3
shc r5, -4
shc r6, -5
shc r7, -6
shc r6, -7
shc r5, -8
shc r4, -9
shc r3, -10
shc r2, -11
shc r1, -12
shc r0, -13
shc r1, -14
shc r2, -15
shc r3, -0
