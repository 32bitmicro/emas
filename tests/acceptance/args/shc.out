@ 0x0000 : 0xe811  /  111 010 0 000 010 001  /  59409
@ 0x0001 : 0xe852  /  111 010 0 001 010 010  /  59474
@ 0x0002 : 0xe893  /  111 010 0 010 010 011  /  59539
@ 0x0003 : 0xe8d4  /  111 010 0 011 010 100  /  59604
@ 0x0004 : 0xe915  /  111 010 0 100 010 101  /  59669
@ 0x0005 : 0xe956  /  111 010 0 101 010 110  /  59734
@ 0x0006 : 0xe997  /  111 010 0 110 010 111  /  59799
@ 0x0007 : 0xebd0  /  111 010 1 111 010 000  /  60368
@ 0x0008 : 0xeb91  /  111 010 1 110 010 001  /  60305
@ 0x0009 : 0xeb52  /  111 010 1 101 010 010  /  60242
@ 0x000a : 0xeb13  /  111 010 1 100 010 011  /  60179
@ 0x000b : 0xead4  /  111 010 1 011 010 100  /  60116
@ 0x000c : 0xea95  /  111 010 1 010 010 101  /  60053
@ 0x000d : 0xea56  /  111 010 1 001 010 110  /  59990
@ 0x000e : 0xea17  /  111 010 1 000 010 111  /  59927
@ 0x000f : 0xe850  /  111 010 0 001 010 000  /  59472
@ 0x0010 : 0xea97  /  111 010 1 010 010 111  /  60055
@ 0x0011 : 0xead6  /  111 010 1 011 010 110  /  60118
@ 0x0012 : 0xeb15  /  111 010 1 100 010 101  /  60181
@ 0x0013 : 0xeb54  /  111 010 1 101 010 100  /  60244
@ 0x0014 : 0xeb93  /  111 010 1 110 010 011  /  60307
@ 0x0015 : 0xebd2  /  111 010 1 111 010 010  /  60370
@ 0x0016 : 0xeb91  /  111 010 1 110 010 001  /  60305
@ 0x0017 : 0xeb50  /  111 010 1 101 010 000  /  60240
@ 0x0018 : 0xe917  /  111 010 0 100 010 111  /  59671
@ 0x0019 : 0xe8d6  /  111 010 0 011 010 110  /  59606
@ 0x001a : 0xe895  /  111 010 0 010 010 101  /  59541
@ 0x001b : 0xe854  /  111 010 0 001 010 100  /  59476
@ 0x001c : 0xe813  /  111 010 0 000 010 011  /  59411
@ 0x001d : 0xe852  /  111 010 0 001 010 010  /  59474
@ 0x001e : 0xe891  /  111 010 0 010 010 001  /  59537
@ 0x001f : 0xe8d0  /  111 010 0 011 010 000  /  59600
